src/geo/src/GeoPMTFactoryBase.cc

```diff
@@ -120,6 +120,33 @@ namespace RAT {
         int    light_cone       = table->GetI("light_cone");
         bool   lightcones = false;
         if( light_cone == 1 ){ lightcones = true; G4cout << "Light cones are added!! \n "; }
+        //material properties
+        G4Material* light_cone_material = G4Material::GetMaterial("TiO2");
```

Comment:
For default, lets go with something like tyvek (because the optics in OPTICS.ratdb currently has TiO2 as a dielectric-dielectric interface; which ignores reflectivity and uses snells law combined with surface roughness to transport photons).

Reply:
ok, the swtich to tyvek sounds good. also, the default cone should line up with the pmt face (should be attached slightly above the pmt equator). can i adjust this in GeoPMTFactoryBase.cc?

Reply:
You could move PMTConstructionParams above this light cone code, and then instead of of setting the default light_cone_innerradius = 12.65, you could get the information directly from the PMT, something like
```
double light_cone_innerradius = *std::max_element( (pmtParam.rhoEdge).begin(), (pmtParam.rhoEdge).end() );
```